web/src/components/netflow-traffic.tsx

```diff
@@ -453,9 +454,32 @@ export const NetflowTraffic: React.FC<NetflowTrafficProps> = ({
             model.setStats(stats);
           })
           .catch(err => {
+            const errStr = getHTTPErrorDetails(err, true);
+
+            // check if it's a prom missing label error
+            if (isPromMissingLabelError(errStr)) {
+              let filtersDisabled = false;
+              model.filters.list.forEach(filter => {
+                const fieldName = model.config.columns.find(col => col.filter === filter.def.id)?.field;
+                if (!fieldName || errStr.includes(fieldName)) {
+                  filtersDisabled = true;
+                  filter.values.forEach(fv => {
+                    fv.disabled = true;
+                  });
+                }
+              });
+              if (filtersDisabled) {
+                // update filters to retrigger query without showing the error
+                updateTableFilters({ ...model.filters });
+                model.setChipsPopoverMessage(getPromError(errStr));
```

Comment:
still have an issue here when a topology group is enabled. This will loop as the filters are not involved

Reply:
that would make it but a bit hacky:
https://github.com/netobserv/network-observability-console-plugin/pull/662/commits/a1cbc9bc4a8d484072dacabee242e3febae451f3

The positive side is that it will manage any case (not only the topology groups)